src/main/scala/org/ergoplatform/nodeView/mempool/OrderedTxPool.scala

```diff
@@ -162,7 +164,8 @@ object OrderedTxPool {
       .map(_.value)
       .sum
     // We multiply by 1024 for better precision
-    WeightedTxId(tx.id, fee * 1024 / tx.size)
+    val feePerKb = fee * 1024 / tx.size
+    WeightedTxId(tx.id, feePerKb, feePerKb, System.currentTimeMillis())
```

Comment:
The time source can be abstacted using time provider, then the `OrderedTxPool.put` method can be unit-tested (where this method is used).